scripts/schema_linter.py

```diff
@@ -2,30 +2,34 @@
 
 """Script to employ linkml-linter"""
 import subprocess
+import sys
 from pathlib import Path
 
 
 HERE = Path(__file__).parent.resolve()
 LINTER_CONFIG = HERE.parent / ".linkml_linter.yaml"
 SCHEMA_DIR = HERE.parent / "src" / "schema"
-LINTING_DOC = HERE.parent / "docs" / "linkml_linter.md"
 
 
-def run_linter(linkml_yaml: Path, report: Path):
+def run_linter(schema_yaml: Path):
     """Function to call linkml-linter"""
-    return subprocess.Popen(
-        [
-            "linkml-lint",
-            "--config",
-            LINTER_CONFIG,
-            "--format",
-            "markdown",
-            "--output",
-            str(report),
-            str(linkml_yaml),
-        ]
-    )
+    try:
+        subprocess.run(
+            [
+                "linkml-lint",
+                "--config",
+                LINTER_CONFIG,
+                str(schema_yaml),
+            ],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            encoding="utf-8",
+            check=True,
+        )
+    except subprocess.CalledProcessError as exc:
+        print(f"Status : FAIL, {exc.returncode}, {exc.stdout}")
+        sys.exit(1)
 
 
 if __name__ == "__main__":
-    run_linter(SCHEMA_DIR, LINTING_DOC)
+    run_linter(SCHEMA_DIR)
```

Comment:
Might be nicer to raise an exception in the run_linter. A main function could be added that just calls the run_linter and translates the exception into a nicely colored output.